### DISCUSSION TITLE
Production mode issue

### DISCUSSION THREAD

**Discussion by salmandev1** [vercel/community#4009] [Help] [answered]
### Question

I am using npm run start to run project in production mode after that if i do any change in project no change will appear in browser url, now i need a solution if i do any change in project that change will appear in browser without stop the next js production mode. can you please help

**Comment by amyegan** [comment, marked as answer]
Hey @salmandev1. Any code change you make will require a new deployment. Content changes can be made without a new deployment for sites that use [ISR](https://nextjs.org/docs/pages/building-your-application/data-fetching/incremental-static-regeneration).

If you let me know what kind of changes you need to make, I can help you find the resources you need.
